.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+
+node_js:
+  - "stable"
+
```

Comment:
add this `env:
  global:
    - NODE_ENV=test` learn more here [Here](https://docs.travis-ci.com/user/environment-variables/#global-variables)